suite-common/icons/assets/icons/accounts.svg

```diff
@@ -1,3 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" fill="none" viewBox="0 0 24 24">
-  <path stroke="#1B1A19" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5" d="M19.125 4.125H4.875a.75.75 0 0 0-.75.75v14.25c0 .414.336.75.75.75h14.25a.75.75 0 0 0 .75-.75V4.875a.75.75 0 0 0-.75-.75m-7.125 0v15.75M12 7.5h7.875M12 10.5h7.875M12 13.5h7.875M12 16.5h7.875"/>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" fill="none" viewBox="0 0 24 24"><path d="M4.348 3.479 C 3.964 3.629,3.624 3.973,3.477 4.358 C 3.402 4.556,3.400 4.755,3.400 12.000 C 3.400 19.245,3.402 19.444,3.477 19.642 C 3.625 20.032,3.958 20.365,4.368 20.533 L 4.580 20.620 12.000 20.620 L 19.420 20.620 19.632 20.533 C 20.042 20.365,20.365 20.042,20.533 19.632 L 20.620 19.420 20.620 12.000 L 20.620 4.580 20.533 4.368 C 20.365 3.958,20.032 3.625,19.642 3.477 C 19.444 3.402,19.247 3.400,11.990 3.402 C 4.801 3.404,4.533 3.406,4.348 3.479 M11.240 12.000 L 11.240 19.120 8.060 19.120 L 4.880 19.120 4.880 12.000 L 4.880 4.880 8.060 4.880 L 11.240 4.880 11.240 12.000 M19.120 5.820 L 19.120 6.760 15.940 6.760 L 12.760 6.760 12.760 5.820 L 12.760 4.880 15.940 4.880 L 19.120 4.880 19.120 5.820 M19.120 9.000 L 19.120 9.760 15.940 9.760 L 12.760 9.760 12.760 9.000 L 12.760 8.240 15.940 8.240 L 19.120 8.240 19.120 9.000 M19.120 12.000 L 19.120 12.760 15.940 12.760 L 12.760 12.760 12.760 12.000 L 12.760 11.240 15.940 11.240 L 19.120 11.240 19.120 12.000 M19.120 15.000 L 19.120 15.760 15.940 15.760 L 12.760 15.760 12.760 15.000 L 12.760 14.240 15.940 14.240 L 19.120 14.240 19.120 15.000 M19.120 18.180 L 19.120 19.120 15.940 19.120 L 12.760 19.120 12.760 18.180 L 12.760 17.240 15.940 17.240 L 19.120 17.240 19.120 18.180 " stroke="none" fill="black" fill-rule="evenodd"></path></svg>
```

Comment:
are those svgs optimised? Each that I checked looks much more complex than before

Reply:
I tried one randomly in https://svgoptimizer.com/ and it shaked 30% of bytes

Reply:
We converted all icons to be implemented without stroke. All major design systems like MUI are using it this way.

There are multiple issues with stroke:

1) you can't change color of the icon properly
2) you can't scale properly

![image](https://github.com/oslllo/svg-fixer/raw/master/docs/images/demo.gif)